q2_metadata/templates/tabulate/index.html

```diff
@@ -3,22 +3,59 @@
 {% block content %}
   <div class="row">
     <div class="col-lg-12">
-      <p>Export options:</p>
-      <table id='table' class='table table-hover'></table>
+      <p>
+        <a href="metadata.tsv" target="_blank" rel="noopener noreferrer" class="btn btn-default">
+          Download normalized (and if applicable, merged) metadata
```

Comment:
What do you mean by "normalized" metadata in this context?

Can you include the format in the button text, e.g. "Download metadata TSV file" ?

Reply:
I was thinking that it might be nice to somehow indicate that the metadata file you get back out of this viz might not be 100% the same as what you put in, aside from merging. In particular, the addition of the `types` directive row, as well as any whitespace stripping. I am fine with dropping "normalized" here.

Reply:
I will also drop the parenthetical, since that seems to be implied, too.